components/script/dom/webidls/MutationObserver.webidl

```diff
@@ -0,0 +1,18 @@
+[Pref="dom.mutation_observer.enabled", Constructor(MutationCallback callback)]
+interface MutationObserver {
+  void observe(Node target, optional MutationObserverInit options);
+  void disconnect();
+  sequence<MutationRecord> takeRecords();
```

Comment:
Let's comment out all of these methods in this file until we actually implement them.